plugins/kafka/src/components/ConsumerGroupOffsets/ConsumerGroupOffsets.tsx

```diff
@@ -104,7 +104,7 @@ export const ConsumerGroupOffsets = ({
           <Typography variant="h6">
             Consumed Topics for {consumerGroup} (
             {(dashboardUrl && (
-              <Link href={dashboardUrl} target="_blank">
+              <Link to={dashboardUrl} target="_blank">
```

Comment:
A final request: could you go through all of these changed rows, and remove all usage of `target` and `rel`? One of the good things about our Link is that it standardizes all of that behavior.

Reply:
👍  done